edgemesh/pkg/plugin/registry/registry.go

```diff
@@ -141,7 +140,9 @@ func parseServiceURL(serviceURL string) (name, namespace string, port int, err e
 	} else {
 		klog.Errorf("[EdgeMesh] service url %s invalid", serviceURL)
 		err = fmt.Errorf("service url %s invalid", serviceURL)
+		return
```

Comment:
```suggestion
		return fmt.Errorf("service url %s invalid", serviceURL)
```
And the error in L137 also need return :)

Reply:
It is not necessary. In L129, the function definition showed the return values.

Reply:
We can delete the unneeded return values.

Reply:
fixed